reledmac.dtx

```diff
@@ -6032,6 +6115,25 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}{\leftlinenumannotation}%
+% \begin{macro}{\rightlinenumannotation}%
+% The \cs{leftlinenumannotation} and \cs{rightlinenumannotation} print the line number annotation, in the case user defined it must be printed in the opposite side regarding the main line number.
```

Comment:
I *think* this should say: "in the user-defined case it must be printed on the side opposite the main line number".

Reply:
maybe simplier "when user defined it must be printed..." ?

Reply:
It should also say "prints the line number annotation", sorry I didn't spot that before; change the comma after "annotation" to a semi-colon. And I would keep the hyphen in "user-defined". So in total:

> prints the line number annotation; when user-defined it must be printed on the side opposite the main line number.

Reply:
done!